src/controllers/_authentication_controllers.py

```diff
@@ -21,11 +23,53 @@ def login_handler():
         )
 
         if result.auth is not None:
-            jwt = result.auth.token
+            jwt = result.auth.token  # noqa: F811
```

Comment:
Seems that the comment isn't needed, anyways, avoid using comments to ignore linter warnings instead of fixing them.